### ISSUE TITLE
BUG: EX Portal Application Free-Form on Project Connect (On PROD🎉🎉)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#8926]
### The portal application page that collects the free-form data is displaying the free-form rows for TURP and SCP.
<img width="741" alt="Screen Shot 2022-04-15 at 2 24 47 PM" src="https://user-images.githubusercontent.com/49930868/163626241-ef73eb71-d9b2-4a1c-aee1-b742575da9fd.png">

<img width="811" alt="Screen Shot 2022-04-15 at 2 26 37 PM" src="https://user-images.githubusercontent.com/49930868/163626263-2c96d841-c810-4e93-9733-670df3e980ce.png">

**Comment by SubhaShroff** [comment]
@TracyLinder  @Nadin-Nader  Checked for this bug on DEV & TEST Portals. DEV portal doesn't have this bug. Not sure why TEST/PROD portals are not in sync with DEV. I have sent an email to Swapna(CCed to you both) about it.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  This bug was fixed in 6844 and the commit was [82377d878650](http://coaamandadevops01.austintexas.gov/R3:82377d878650982dbfa423f6c3cbd436c98854d1). 6844 was on PROD. The related lookup table=50226 entries are not in TEST/PROD now. Not sure, how those entries got reverted.
I will get in contact with CTM team and see if I need to redo the commit or if they can use the same old commit to fix this issue in TEST/PROD.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  DEV deployment is done. I tested it on DEV portal for extra fields. As it was already working(not displaying the extra freeform fields on DEV) and no change after the deployment. So, after you have approved, I submitted TEST promotion request for this ticket - SCTASK0214398.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  This is deployed in TEST. I tested it on TEST Portal. The extra fields are removed. Here is the screenshot of it.

![image](https://user-images.githubusercontent.com/51680715/165397332-1c12d069-cac0-417c-bf99-4f0be04088e0.png)

![image](https://user-images.githubusercontent.com/51680715/165397443-eba42e2b-aabe-40c2-bce9-beb64cacae18.png)

I think its ready for UAT.

**Comment by SubhaShroff** [comment]
On Friday ROW Amanda DTS call, Paloma has mentioned about project Information info group help text displaying twice on Portal. I am fixing the bug in this ticket particularly, for EX.
_NOTE: I noticed within other ROW permits also, Project Information group scattered and with each chunk, the respective help text repeating on Portal. It needs to be fixed for each permit type. If we can have separate ticket for the other permits, that would be good._

**Comment by SubhaShroff** [comment]
@Nadin-Nader  The above mentioned bug on portal should be fixed with this new commit -"[2cbc49532637](http://coaamandadevops01.austintexas.gov/R3:2cbc495326371315028c8fa2a1a09457b3cb47d5)"
I submitted a DEV promotion request - SCTASK0215947 for the commit.

**Comment by Nadin-Nader** [comment]
@SubhaShroff this has been deployed to DEV. please UAT

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I had tested this yesterday, all the project information related infos displaying under one group for EX PC in back office. But the portal still looked the same. Today, I had requested Swapna to refresh the Portal Server and that has set the things right. 
Now on DEV portal also, Project Information infos are not divided into sections and so the help text. Looks good to go to TEST.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0216749 - is submitted for TEST promotion to fix the Project Information Help text duplicating.
_(Actual issue is fixed that Project Information group was splitting. Now all infos related to this group are under one section.)_

**Comment by Nadin-Nader** [comment]
@SubhaShroff this has been deployed to TEST. Please UAT

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I tested this on TEST, as usual it needs TEST Portal refresh. Requested Swapna to refresh. Once its done will test it.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  This is done on TEST and I tested it. Looks good. Ready for UAT.

**Comment by Nadin-Nader** [comment]
Waiting on Paloma or ROW to approve the move to PROD

**Comment by Nadin-Nader** [comment]
@SubhaShroff got the approval to move to PROD. Go ahead and send it

**Comment by SubhaShroff** [comment]
@Nadin-Nader SCTASK0218889 - PROD deployment request is submitted.

**Comment by SubhaShroff** [comment]
@Nadin-Nader @TracyLinder   PROD deployment is done.

**Comment by Nadin-Nader** [comment]
This is good to close
